server/rest/src/main/java/org/infinispan/rest/resources/ServerResource.java

```diff
@@ -54,6 +53,7 @@ public Invocations getInvocations() {
             .invocation().methods(GET).path("/v2/server/env").handleWith(this::env)
             .invocation().methods(GET).path("/v2/server/memory").handleWith(this::memory)
             .invocation().methods(GET).path("/v2/server/stop").handleWith(this::stop)
+            .invocation().methods(GET).path("/v2/server/stop-cluster").handleWith(this::stopCluster)
```

Comment:
I was thinking in having REST operations that involves multiple server under a separate resource: /rest//v2/cluster and keep the server resource to target each individual server. @karesti has also some ops that need to affect all servers to gather info.

https://issues.jboss.org/browse/ISPN-10580

Reply:
@karesti I'll introduce the ClusterResource in this PR

Reply:
shouldn't this be removed now?

Reply:
Well spotted. Repushed.